react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+// standard
+const listItems = pokedex.map(item =>
+  <li>{item.name}</li>
```

Comment:
Be sure to also include a `key` prop for each `li` element.

Reply:
Keys added: 41be93d